lms/services/event.py

```diff
@@ -41,7 +41,13 @@ def insert_event(self, event: BaseEvent):
     @lru_cache(maxsize=10)
     def _get_type_pk(self, type_: EventType.Type) -> int:
         """Cache the PK of the event_type table to avoid an extra query while inserting events."""
-        return self._db.query(EventType).filter_by(type=type_).one().id
+        event_type = self._db.query(EventType).filter_by(type=type_).one_or_none()
```

Comment:
Creating new event types as we see them.

The alternative is doing a migration but I reckon as this is all based on the `Type` enum it's fine for python to control the DB state as well.